### DISCUSSION TITLE
v2.7.376

### DISCUSSION THREAD

**Discussion by This-is-XiaoDeng** [Moonlark-Dev/XDbot2#189] [Announcements]
## What's Changed
* 同步Develop分支 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/162
* Develop by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/164
* 添加插件：XDbot统计信息 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/165
* 修复已知问题提 & 添加插件：邀请加群自动同意 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/166
* [fix] 修复使用部分语言时无法匹配回复Cave的问题 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/167
* Update rule.py by @montmorill in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/171
* 增加了tips by @YifanZhuZhu in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/172
* Develop by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/173
* [update] issue模板 by @xxtg666 in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/175
* [add] 添加监听群聊提示  &  [fix] 修复错误记录会将正常上报内容计入的问题 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/174
* [update] 修改邀请加群提示 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/177
* [update] 修改非 su 邀请加群时的提示 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/179
* [update] 格式化快速微积分输出/回答 by @xxtg666 in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/181
* 解决 #178 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/182
* 添加插件：速算平均用时 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/183
* [update] 允许在 cave-d 查看图片数量 by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/184
* 添加语言：zh_hk by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/187
* [update] 版本更新：v2.7.376  by @This-is-XiaoDeng in https://github.com/ITCraftDevelopmentTeam/XDbot2/pull/188

## 更新日志

1. 修复已知问题
2. 添加插件：统计信息
3. 将 report 改为 反馈
4. 将 GPT 使用 API 由 SteamShip 更改为 OpenAI
5. 修改部分本地化键名
6. 在 reply 保存图片时仅保存图片对应 URL
7. Rule V3 （尚未完成）
8. 随机发图时有一定概率直接发送审核库图片
9. 整理部分代码
10. 添加了一些 tips
11. 减小 quickmath 题目发送间隔
13. 添加群聊被监听提示
14. 修改 issues 模板
15. 非 su 邀请加群不再自动同意
16. 在 GPT、CHATGPT 发送消息时回复原消息
17. 更新 XDbot2 依赖列表
18. 格式化 快速微积分 输出
19. 添加插件：平均速算用时
20. 在触发 GPT 且有回复时自动回复的消息一并提交给 ChatGPT
21. 允许在 cave-d 中查看图片数量（需要更新语言）
22. 添加语言：zh_hk

**Full Changelog**: https://github.com/ITCraftDevelopmentTeam/XDbot2/compare/v2.7.282...v2.7.376

<hr /><em>This discussion was created from the release <a href='https://github.com/ITCraftDevelopmentTeam/XDbot2/releases/tag/'>v2.7.376</a>.</em>

**Comment by YifanZhuZhu** [comment]
支持~
